terraform/modules/ec2_autoscaling_group/locals.tf

```diff
@@ -0,0 +1,77 @@
+locals {
+  name_split = split("-", var.name)
+
+  default_tags = {
+    server-type = join("-", slice(local.name_split, 1, length(local.name_split)))
```

Comment:
I guess this is also quite nomis/oasys specific, I wonder if we should just explicitly define this as well as the nomis-environment